terraform-infrastructure-pipelines/setup-consul-workspace.md

```diff
@@ -30,20 +28,20 @@ Select "Include submodules on clone" then click on "Update VCS settings". This w
 Next, click on "Variables". Your Terraform Variables should already be set for you. These correspond with the variables in [`variables.tf`](https://github.com/hashicorp/learn-terraform-pipelines-consul/blob/master/variables.tf). Verify your variables have been set correctly.
 
 #### Terraform Variables
-- **namespace** — Kubernetes Namespace to deploy the Consul Helm chart<br/>
-  This should be set to `hashicorp-learn`. You will use this to access your Consul and Vault instances later.
-- **organization** - Organization of workspace that created the Kubernetes cluster<br/>
-  This should be set to `infrastructure-pipelines-workshop`.
 - **release_name** — Helm Release name for Consul chart<br/>
   This should be set to `hashicorp-learn`. Your Vault pods will start with this release name.
+- **namespace** — Kubernetes Namespace to deploy the Consul Helm chart<br/>
+  This should be set to `hashicorp-learn`. You will use this to access your Consul and Vault instances later.
 - **cluster_workspace** — Workspace that created the Kubernetes cluster<br/>
-  This should be set to `{firstName}-{lastInitial}-k8s` (`john-d-k8s`).
+  If you didn't customize your workspace name, this is `learn-terraform-pipelines-k8s`.
+- **organization** - Organization of workspace that created the Kubernetes cluster<br/>
```

Comment:
I forget, but did we have users create a new organization for the guide? If not this may be their own org name?

Reply:
Nope, that wasn't a hard requirement

![image](https://user-images.githubusercontent.com/7556325/85610021-76aa7700-b60b-11ea-826d-2d0bfe9e0ac1.png)